### PR TITLE
21815 - kwarg wranglin'

### PR DESCRIPTION
**Pull request by jjf130**
this might be hot garbage

### CODE REVIEW

File: falcano/model.py [hunk 1]
```diff
@@ -567,43 +571,32 @@ def query(cls,
...
-            # Use #A1, #A2, ... to avoid reserved word conflicts
-            names = {f'#A{i}': attr for i, attr in enumerate(attributes_to_get)}
-            query['Select'] = 'SPECIFIC_ATTRIBUTES'
-            query['ProjectionExpression'] = ', '.join(list(names.keys()))
-            query['ExpressionAttributeNames'] = names
-        if page_size:
-            query['Limit'] = page_size
+            select = SPECIFIC_ATTRIBUTES
+
+        kwargs: Dict[str, Any] = dict(
```
Comment (erictwalker18) [inline falcano/model.py:584]: Just beautiful to not have the 10,000 if blocks in every method.

File: falcano/model.py [hunk 2]
```diff
@@ -734,12 +727,18 @@ def batch_write(cls, auto_commit: bool = True):
...
         return BatchWrite(cls, auto_commit=auto_commit)
 
+    # def delete(self, condition=None) -> Any:
+    #     kwargs = {}
+    #     table = self.resource().Table(self.Meta.table_name)
+    #     if condition:
+    #         kwargs['ConditionExpression'] = condition
+    #     kwargs['Key'] = self._get_keys()
+    #     return table.delete_item(**kwargs)
+
```
Comment (erictwalker18) [inline falcano/model.py:737]:
```suggestion
```

File: falcano/model.py [hunk 3]
```diff
@@ -750,35 +749,23 @@ def update(self, actions, condition=None):
             raise TypeError('the value of `actions` is expected to be a non-empty list')
 
         kwargs = {
-            RETURN_VALUES: ALL_NEW,
+            'return_values': ALL_NEW,
```
Comment (erictwalker18) [inline falcano/model.py:752]: Use the constants imported. If they have to be lowercase, you can do
```suggestion
            strincase.snake_case(RETURN_VALUES): ALL_NEW,
```

File: falcano/model.py [hunk 4]
```diff
@@ -750,35 +749,23 @@ def update(self, actions, condition=None):
...
-            )
-        if name_placeholders:
-            kwargs[EXPRESSION_ATTRIBUTE_NAMES] = {v: k for k, v in name_placeholders.items()}
-        if expression_attribute_values:
-            kwargs[EXPRESSION_ATTRIBUTE_VALUES] = expression_attribute_values
 
+        kwargs = self.get_operation_kwargs_from_instance(**kwargs)
         # Get the key and put it in kwargs
-        kwargs[KEY] = self._get_keys()
+        #kwargs[KEY] = self._get_keys()
```
Comment (erictwalker18) [inline falcano/model.py:760]:
```suggestion
```

File: falcano/model.py [hunk 5]
```diff
@@ -1015,15 +1092,15 @@ def get_operation_kwargs(
             operation_kwargs[UPDATE_EXPRESSION] = update_expression.serialize(
                 name_placeholders,
                 expression_attribute_values
-            )
+            ) # TODO: handle non-serialized case?
```
Comment (erictwalker18) [inline falcano/model.py:1095]: Is this still todo?

**Review by erictwalker18 — CHANGES_REQUESTED** [review]
Clean up a couple comments, overall doesn't resemble garbage

File: falcano/model.py (hunk 3 above)
Comment (jjf130) [reply falcano/model.py:752]: Nice... want me to do this everywhere?

File: falcano/model.py (hunk 5 above)
Comment (jjf130) [reply falcano/model.py:1095]: After what we learned about serialization last week, I dont think that it is. I will remove.

File: falcano/model.py (hunk 3 above)
Comment (erictwalker18) [reply falcano/model.py:752]: that'd be nice!

**Review by erictwalker18 — APPROVED** [review]
LGTM

**Pull request merged**